### ISSUE TITLE
Ledger not detected - firmware 1.5.5

### ISSUE THREAD

**Issue by JesusTheHun** [hirishh/liskish-wallet#14]
Liskish 2.0.3-beta
Ledger Nano S firmware v1.5.5

I just downloaded the last version of liskish wallet and upgraded my wallet.
My ledger does not appear in the wallet.
I tried to close the wallet, reconnect my ledger, start wallet. And the opposite, disconnect the ledger, start the wallet, reconnect the ledger. The wallet is still missing :/

How can I provide you more information ?

**Comment by sabotagebeats** [comment]
same with 1.4.2

**Comment by hirishh** [comment]
Hello guys,

First of all, please update to last firmware! Liskish 2.0.3-beta is not compatible with FW 1.4.2.

Then please I need to know on which OS. If linux, please remember to install/add udev rules!
Here more info: https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues

Let me know :)

**Comment by karmatta** [comment]
Im using a mac. Same problem. Help please!

**Comment by hirishh** [comment]
Hello Guys,

First of all, be sure to use the latest liskish wallet (2.0.3-beta) and latest nano S/X firmware.
Please provide to me also the OS you are referring.

@karmatta which OSX version? I know there are some trouble with USB connections. Could you try first to open the Lisk app on you ledger and only after open liskish wallet?

Let me know

**Comment by karmatta** [comment]
"Could you try first to open the Lisk app on you ledger and only after open liskish wallet?"
yes! that worked.. duh!

Thanks!

**Comment by toyopilgrim** [comment]
@karmatta 
I have a similar issue on ubuntu 16.04.   

> Then please I need to know on which OS. If linux, please remember to install/add udev rules.
> Could you try first to open the Lisk app on you ledger and only after open liskish wallet?

Though I tried these, what I always get is still No Hadware Wallet detected...
Would there be any else solution for this? Thanks in advance.

**Comment by toyopilgrim** [comment]
@hirishh
Looks like my outdated firmware version prevented it from being connected. After updating the firmware to 1.5.5 on the ledger, it worked!

**Comment by hirishh** [comment]
@toyopilgrim perfect! I'm glad to hear that :)

**Comment by bip72505** [comment]
liskish-wallet-linux-x86_64-2.0.4-beta.AppImage
Ledger Nano S firmware v1.6.0
Linux with udev rules

Hello, 
I just started using the Ledger and it is not detected in your app. I guess the new firmware broke it again. 

Thanks in advance

**Comment by hirishh** [comment]
Hello @bip72505,
Thank you for your report.
Before digging deeper, did you update to the new udev rules?

![image](https://user-images.githubusercontent.com/7662810/69104594-1ea20580-0a69-11ea-822c-d0ed11722d27.png)

Let me know!

**Comment by hirishh** [comment]
I've tested with the new udev rules and is working fine here. 

Feel free to reopen another issue if you still have some!
